import.cpp

```diff
@@ -1403,9 +1403,9 @@ bool cImport::UpdateXMLTVEvent(cEPGSource *Source, sqlite3 *Db, cXMLTVEvent *xEv
 
         std::string ed=shortdesc;
 
-        int reps;
-        reps=pcrecpp::RE("'").GlobalReplace("''",&ed);
-        if (reps)
+        long unsigned int reps = ed.length();
+        std::regex_replace(ed, std::regex("'"), "''");
+        if (reps != ed.length())
         {
```

Comment:
Similar case here. Replaces all (') with ('') (doubles them up). Here the length based check will work as it does not cover several replaces. But if length is used to verify if changes were made, then a "regular string replace" should do.